oort/oort-aws/src/main/groovy/com/netflix/spinnaker/oort/aws/provider/agent/ClusterCachingAgent.groovy

```diff
@@ -554,6 +575,19 @@ class ClusterCachingAgent implements CachingAgent, OnDemandAgent, AccountAware,
     policy
   }
 
+  private static Map flattenAutoScalingGroup(AutoScalingGroup asg) {
+    return [
+      desiredCapacity   : asg.desiredCapacity,
+      minSize           : asg.minSize,
+      maxSize           : asg.maxSize,
+      suspendedProcesses: asg.suspendedProcesses*.processName.sort()
+    ]
+  }
+
+  private static boolean areSimilarAutoScalingGroups(AutoScalingGroup asg1, AutoScalingGroup asg2) {
+    return flattenAutoScalingGroup(asg1) == flattenAutoScalingGroup(asg2)
```

Comment:
Just using a simple heuristic of things that we particular care about being current (capacities and suspended processes).

Anything else worthwhile?